.github/workflows/unittests.yml

```diff
@@ -89,9 +90,9 @@ jobs:
         with:
           python-version: 3.7
       - name: Install dependencies
-        run: pip install tensorflow==${{matrix.tf-version}} larq~=0.9.1 larq_zoo==1.0.b3 tensorflow_datasets==1.3.2 -e . --no-cache
+        run: pip install tensorflow==${{matrix.tf-version}} larq~=0.9.1 larq_zoo==1.0.b3 tensorflow_datasets==1.3.2 packaging --no-cache
```

Comment:
What is this? How is it still installed if there's no `-e .` ?

Reply:
`packaging` is a pip dependency we use in lce to do version comparison.

On linux to properly build the wheel we needed to add [this workaround](https://github.com/larq/compute-engine/blob/master/setup.py#L19) copied from TensorFlow Addons. Unfortunately this prevents us from installing the package without the C++ dependencies. Therefore I add the working directory to the `PYTHONPATH` which removes the need to install LCE via `pip` for testing.